lib/Frontend/DiagnosticVerifier.cpp

```diff
@@ -128,7 +128,7 @@ findDiagnostic(std::vector<CapturedDiagnosticInfo> &CapturedDiagnostics,
 
     // Verify the classification and string.
     if (I->Classification != Expected.Classification ||
-        I->Message.find(Expected.MessageStr) == StringRef::npos)
+        !I->Message.contains(Expected.MessageStr))
```

Comment:
I think this one and similar checks on `StringRef` are not right. In the JIRA, I only noted the change for Set-like types, I don't think there is a similar method on `StringRef`, which is why there is a CI error here. Did you add a `contains` method on `StringRef` in upstream LLVM?

Reply:
When I was working on this, I found that there was `contains` method in [`StringRef`](https://github.com/apple/llvm-project/blob/1044437905e95a42372d7c1c2d341ebc729570ce/llvm/include/llvm/ADT/StringRef.h#L465) so I worked on them as well. 

I think this failed because `I→Message` turned out to be `SmallString` not `StringRef`. I reverted the change here, could you please run the tests again? :)

Reply:
Oh you're right, it's `SmallString` and not `StringRef`. I should not be reviewing code around midnight lol.